src/composables/usePagination.ts

```diff
@@ -0,0 +1,43 @@
+import { type Ref, ref, computed, watch } from 'vue';
+
+export function usePagination(totalPages: Ref<number>, initialItemsPerPage: number, maxPagesToShow?: number) {
```

Comment:
Setting the default value on the function declaration removes the need for the `or` on line 9, also it is shown by VSCode so you can see the default value without reading the code

https://github.com/RedHatProductSecurity/osim/blob/5eda9e9e4fe8ae821210bb9abc9aae58ec169342/src/composables/usePagination.ts#L9


```suggestion
export function usePagination(totalPages: Ref<number>, initialItemsPerPage: number, maxPagesToShow = 7) {
```